spyder_kernels/utils/tests/test_nsview.py

```diff
@@ -56,7 +56,7 @@ def __getattr__(self, name):
 
 
     length = [list([1,2,3]), tuple([1,2,3]), set([1,2,3]), '123',
```

Comment:
```suggestion
    length = [
        list([1, 2, 3]),
        tuple([1, 2, 3]),
        set([1, 2, 3]),
        '123',
```

Let's improve the formatting of this list for the future (I have to leave two suggestions for it).